### ISSUE TITLE
On navigating on the stack bar, focus Initially lands at last bar series and the position of bar series incorrectly numbered.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#17439]
#### Expected behaviour
On navigating on the ‘Stacked bar' focus initially focus should land at first bar series in order to navigate to next bar series user need to press 'right arrow key and
For ex: When focus lands on bar graph focus initially should land at first bar series and Screen reader should announces. High charts interactive chart. region Interactive chart Created with Highcharts 10.1.0 John, bar series 1 of 3 with 5 bars. region Apples. 3.Joe ,graphic

#### Actual behaviour
On navigating on the ‘Stacked bar' focus initially lands at last bar series in order to navigate to previous bar series user need to press 'left arrow key and navigate in reverse direction and the position of bar series incorrectly numbered.
For ex: When focus lands on bar graph focus initially land on last bar series Screen reader announces. High charts interactive chart. region Interactive chart  Created with Highcharts 10.1.0 John, bar series 1 of 3 with 5 bars. region Apples. 5.John,graphic

#### Live demo with steps to reproduce
1.Open URL: https://www.highcharts.com/demo/bar-stacked in the Edge browser.
2.Navigate to ‘Stacked bar’ under ‘Column and Bar Charts’ in the left navigation.
3.Navigate to Stacked bar serious and observe the issue.

![On navigating on the bar Focus initially lands at last bar series and the position of bar series incorrectly numbered](https://user-images.githubusercontent.com/93735775/175749077-8ddc03ea-0436-4ebb-ad0d-c172a4321692.png)
[On navigating on the bar Focus initially lands at last bar series and the position of bar series incorrectly numbered.zip](https://github.com/highcharts/highcharts/files/8983372/On.navigating.on.the.bar.Focus.initially.lands.at.last.bar.series.and.the.position.of.bar.series.incorrectly.numbered.zip)

**Comment by msftedad** [comment]
Hi Team, is there any update on this bug?

**Comment by oysteinmoseng** [comment]
Hi @msftedad, the workaround here would be to set [yAxis.reversedStacks](https://api.highcharts.com/highcharts/yAxis.reversedStacks) to `false`. Example: https://jsfiddle.net/oysteinmoseng/gfp3q5w6/ 

We are also looking into more intuitive default navigation behavior for stacked bar charts this year.

**Comment by msftedad** [comment]
@oysteinmoseng Thanks for the update! we shall try this and get back to you.
